TelephoneBook/main.c

```diff
@@ -0,0 +1,125 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <locale.h>
+#include "test.h"
+#include "telephonebook.h"
+
+enum Options {quit, add, print, findPhoneByName, findNameByPhone, save};
```

Comment:
Это обычно в столбик пишут, так:
```suggestion
enum Options {
    quit, 
    add, 
    print, 
    findPhoneByName, 
    findNameByPhone, 
    save 
};
```